src/liboslexec/pointcloud.cpp

```diff
@@ -379,89 +393,76 @@ RendererServices::pointcloud_write(ShaderGlobals* /*sg*/, ustringhash filename,
 
 namespace pvt {
 
```

Comment:
Extract osl_point_* to oppointcould.cpp so bitcode compilation doesn't need to see the implementation above.